addons/load-extensions/addon.json

```diff
@@ -1,6 +1,18 @@
 {
   "name": "Automatically add extensions",
-  "description": "Auto-shows music, pen, and other extensions in the block category menu in the editor.",
+  "description": "Automatically adds the selected block extensions such as music or pen to the block category menu in the editor.",
+  "info": [
+    {
+      "type": "warning",
```

Comment:
Does this need to be a warning? What effects could it have?

Reply:
I could probably reduce it to a notice. I set it to a warning because users probably won't want the camera automatically tuning on so often.

Reply:
> I could probably reduce it to a notice. I set it to a warning because users probably won't want the camera automatically tuning on so often.

Changing it would help with World_Languages' point, and with your reasoning it would be a better notice.